rosette-scala/src/main/scala/coop/rchain/rosette/Main.scala

```diff
@@ -1,7 +1,41 @@
 package coop.rchain.rosette
 
-object Main {
+import java.io.InputStreamReader
 
-  def main(args: Array[String]): Unit =
-    println("Hello, world!")
+import coop.rchain.rosette.parser.bytecode._
+import Show._
+
+object Main extends App {
+
+  /*
+  // TODO: Remove before merge
+  val test = Seq[Int](
+    Integer.parseInt("00000000", 2), // halt
+    Integer.parseInt("00000001", 2), // ignored arg
+    Integer.parseInt("00000011", 2), // nargs
+    Integer.parseInt("00101010", 2), // nargs arg
+    Integer.parseInt("00001011", 2), // outstanding
+    Integer.parseInt("00000000", 2), // outstanding p arg
+    Integer.parseInt("00000000", 2), // outstanding n arg
+    Integer.parseInt("00001000", 2), // outstanding x arg
+  )
+
+  def writeFile(filename: String, bytes: Seq[Int]): Unit = {
+    val bos = new BufferedOutputStream(new FileOutputStream(filename))
+    bos.write(bytes.map(_.toByte).toArray)
+    bos.close()
+  }
+
+  writeFile("test.txt", test)
+   */
+
+  val bytes = collection.mutable.ArrayBuffer[Int]()
+  val stdin = new InputStreamReader(System.in)
+
+  Stream.continually(stdin.read()).takeWhile(_ != -1).foreach(bytes += _)
```

Comment:
How does this compare to `java.nio.file.Files.readAllBytes`?

Reply:
I think we can't read from `stdin` with `java.nio.file.Files.readAllBytes`. Only from files.